File: flask/app/models/usergroupstate.py
from app import db


class Usergroupstate(db.Model):
    __tablename__ = 'usergroupstate'
    id = db.Column(db.Integer, primary_key=True)
    label = db.Column(db.String, unique=True, nullable=False)

    # groups = db.relationship("Group", back_populates="state")
    # users = db.relationship("User", back_populates="state")

    def __init__(self, **kwargs):
        super().__init__(**kwargs)

File: express/models/UserGroupState.js
const { Sequelize, DataTypes, Model } = require('sequelize');
const sequelize = require('../db/db');

class Usergroupstate extends Model {}
Usergroupstate.init({
    id: {
      type: DataTypes.INTEGER,
      autoIncrement: true,
      primaryKey: true,
    },
    label: {
      type: DataTypes.STRING,
      allowNull: false,
      unique: true,
    },
  }, {
    sequelize,
    modelName: 'Usergroupstate',
    tableName: 'usergroupstate',
    underscored: true,
    timestamps: false
  })

module.exports = Usergroupstate;
